quotes.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+	<html>
+	<head>
+		<link rel="stylesheet" type="text/css" href="css/styles.css">
+		<title>KANYE RESUME</title>
+	</head>
+
+	<body>
+		<nav></nav>
+		<nav></nav>
+			
+			<div class="header">
+				<h1>KANYE WEST</h1>
+
+				<ul>
+					<li style="color: black">QUOTES</li>
+					<li style="color: black"><b>||</b></li>
+					<li>
+						<a href="index.html">ME, MYSELF & I
+					</li> 
+				</ul>
+			</div>
+		
+		<nav></nav>
+
+			<div>
+					<h5>'MY GREATEST PAIN IN LIFE IS THAT I WILL NEVER BE ABLE TO SEE MYSELF PERFORM LIVE.'</h5>
+			</div>
+
+			<div class="quote">
+					<img src="images/quote1.png" alt="I">
+			</div>
+
+			<div>
+					<h5>'I AM WARHOL. I AM SHAKESPEARE IN THE FLESH. WALT DISNEY. NIKE. GOOGLE.'</h5>
+			</div>
+
+			<div class="quote">
+					<img src="images/quote2.png" alt="I">
+			</div>
+
+			<div>
+					<h5>'I STILL THINK I AM THE GREATEST.'</h5>
+			</div>
+
+		<nav></nav>
+
+
+			<div class="photoblock">
```

Comment:
@humphriescarly Very creative use of the .photoblock and .photoblank classes & associated styling. Looks great on the page!